template.typ

```diff
@@ -33,21 +33,20 @@
 
   set text(
     font: "Libertinus Serif",
-    lang: "it",
-    region: "it",
+    lang: locale,
+    region: locale,
     size: 12pt,
     hyphenate: true,
   )
 
-  // Bozza header
   set page(header: if not final {
     box(
       width: 100%,
       stroke: 1pt + red,
       inset: .5em,
       align(
         center + horizon,
-        text(fill: red, size: 18pt, [Bozza del #datetime.today().display("[day]/[month]/[year]")]),
+        text(fill: red, size: 18pt, [#i18n.at(locale).draft_dated #datetime.today().display("[day]/[month]/[year]")]),
```

Comment:
We could add a field for date display in `i18n.typ`

Reply:
Sure! I'll add an entry to specify the date format string. Do you have any specific format in mind for the en locale? en-uk should be the same of it locale, but en-us would have month and day swapped, but I didn't want to go this deeper though.

Reply:
I was thinking about the US format, but I don't think we need it right now so no problems